build.sbt

```diff
@@ -164,7 +158,7 @@ lazy val enumeratumPlayJson = Project(id = "enumeratum-play-json",
                                       settings = commonWithPublishSettings)
   .settings(testSettings: _*)
   .settings(
-    version := s"1.5.13-SNAPSHOT",
+    version := Versions.Core.head,
```

Comment:
I think a bunch of these `version := Versions.Core.head` can be reverted as you've mentioned.